api/handlers_e2e_test.go

```diff
@@ -1147,6 +1147,100 @@ func TestVersion(t *testing.T) {
 	require.Equal(t, response.Version, "(unknown version)")
 }
 
+// TestAccountsOnlineOnlyParam exercises the `online-only` parameter in `GET /v2/accounts`.
+func TestAccountsOnlineOnlyParam(t *testing.T) {
+
+	db, shutdownFunc := setupIdb(t, test.MakeGenesis())
+	defer shutdownFunc()
+
+	//txn := transactions.SignedTxnWithAD{
```

Comment:
We should remove the commented out code prior to merging.

Reply:
Removed in https://github.com/algorand/indexer/pull/1646/commits/63b444f2c479cb1bb15562223905b27277392dfa.

I intially wrote those comments because some of the tests in the file have them - I thought it was some kind of unwritten convention. Turned out it wasn't.